### PR TITLE
Add documentation for Alpha3 Pump component

### PR DESCRIPTION
**Pull request by jan-hofmeier**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#3787

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/alpha3.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- **head** (*Optional*): The estimated head (preasure difference generated by the pump) in ``m``.
+  - **name** (**Required**, string): The name for the head sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **power** (*Optional*): The power consumption of the pump in ``W``.
+  - **name** (**Required**, string): The name for the power sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **speed** (*Optional*): the speed of the pump rotor in ``RPM``.
+  - **name** (**Required**, string): The name for the speed sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
```
Comment (jesserockz) [inline components/sensor/alpha3.rst]: There seem to be some missing sensors here that are in the code?

**Review by jesserockz — CHANGES_REQUESTED** [review]

File: components/sensor/alpha3.rst (hunk 1 above)
Comment (jan-hofmeier) [reply components/sensor/alpha3.rst]: I added the sensors to the doc

**Comment by jan-hofmeier** [comment]
This is not stale, I am waiting for a review...

File: components/sensor/alpha3.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **power** (*Optional*): The power consumption of the pump in ``W``.
+  - **name** (**Required**, string): The name for the power sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **voltage** (*Optional*): Voltage in ``V``.
+  - **name** (**Required**, string): The name for the Voltage sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **voltage** (*Optional*): Current in ``A``.
+  - **name** (**Required**, string): The name for the Current sensor.
+  - All other options from :ref:`Sensor <config-sensor>`
```
Comment (jesserockz) [inline components/sensor/alpha3.rst]:
```suggestion
- **flow** (*Optional*): The estimated volume flow of the medium in ``m³/h``.
  All options from :ref:`Sensor <config-sensor>`.
- **head** (*Optional*): The estimated head (preasure difference generated by the pump) in ``m``.
  All options from :ref:`Sensor <config-sensor>`.
- **speed** (*Optional*): the speed of the pump rotor in ``RPM``.
  All options from :ref:`Sensor <config-sensor>`.
- **power** (*Optional*): The power consumption of the pump in ``W``.
  All options from :ref:`Sensor <config-sensor>`.
- **voltage** (*Optional*): Voltage in ``V``.
  All options from :ref:`Sensor <config-sensor>`.
- **current** (*Optional*): Current in ``A``.
  All options from :ref:`Sensor <config-sensor>`
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

**Review by jesserockz — APPROVED** [review]

**Pull request merged**